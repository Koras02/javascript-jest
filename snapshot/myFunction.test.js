const formatUser = require("./myFunction");

test("formats user data correctly", () => {
  const user = { name: "Andrew", age: 35 };
  const result = formatUser(user);

  // 스냅샷 저장
  expect(result).toMatchSnapshot();
});
