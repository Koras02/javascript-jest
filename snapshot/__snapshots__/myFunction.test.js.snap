// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formats user data correctly 1`] = `"User: Andrew, Age: 35"`;
